external_services/lti.py

```diff
@@ -21,7 +25,33 @@ def __init__(self, service, user, instance, host, title, context_id=None, link_i
         link_id = link_id or "aplus{:d}".format(service.pk)
         title = title or link_id
 
-        student_id = self.external_student_id(user)
+        # Gather user information
```

Comment:
Can we have `if  service.allow_real_user_data:` here and then `else` in place of `if not ...`?